pkgs/desktops/gnome/misc/gnome-flashback/default.nix

```diff
@@ -6,6 +6,7 @@
 , gnome-bluetooth
 , gnome-desktop
 , gnome-panel
+, gnome-flashback # We need this knot-tying for mkSessionForWm.
```

Comment:
This will be shadowed by the let binding won’t it?

Reply:
Oh yeah I forgot about that, fixed.